### PR TITLE
helpers.web refactor

### PR DESCRIPTION
**Pull request by l2isbad**
Changes:

1.
```go
	Header   map[string]string `yaml:"headers"`
	Username string            `yaml:"username"`
	Password string            `yaml:"password"`
```
were moved to `RawRequest` struct.

2.  `Client` defaults:
 * `InsecureSkipVerify` `true` => `false`
 * `FollowRedirect` `false` => `true`

### CODE REVIEW

File: internal/pkg/helpers/web/web.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package web
+
+// Web is a struct with embedded RawRequest and RawClient.
+type Web struct {
```
Comment (Wing924) [inline internal/pkg/helpers/web/web.go]: `Web` name is not so good.
How about `HTTP`?
cf. teregraf use http: https://github.com/influxdata/telegraf/tree/master/plugins/inputs/http

**Comment by l2isbad** [comment]
What about `InsecureSkipVerify` i feel like we don't even need such option

**Comment by Wing924** [comment]
InsecureSkipVerify is very very useful.
Now a days, some apps force you to use https :(
Because many companies don't have in house PKI, they use self-signed certs...

**Comment by Wing924** [comment]
Btw, I think `InsecureSkipVerify` should be `false` by default, but we need to provide a option to enable it.

**Comment by l2isbad** [comment]
Ok, i think this is ready for now. @Wing924

**Pull request merged**